Automattic/woocommerce-services
# Fix tax recalculation on admin order screen missing street address.

Fixes #2039 

To test:
- Follow replication steps on https://github.com/Automattic/woocommerce-services/issues/2039
   - With `22325 W 47th St, Shawnee, Kansas, US 66226` as the origin and destination address, the tax rate should be 9.6%
- Confirm that `woocommerce-services-admin-order-taxjar` javascript file loads on new order page and edit order page
- Confirm that `Recalculate` works correctly for new admin order and an existing admin order
